pkg/vpcmodel/grouping.go

```diff
@@ -452,13 +452,15 @@ func (g *GroupConnLines) computeGroupingForDiff() error {
 }
 
 // get the grouped connectivity output
-func (g *GroupConnLines) String() string {
+func (g *GroupConnLines) String() (string, bool) {
```

Comment:
few of the `String()` functions here are modified to return `(string, bool)` for indication of the statefuleness property.. let's instead try to split to another function such as `HasUnidirectionalConns()` or `HasStatelessConns()` returning this `bool` value, and keep the `String()` function return only `string`